src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -113,9 +114,12 @@ const QuickMarcEditor = ({
   const confirmationChecks = useRef({ ...REQUIRED_CONFIRMATIONS });
   const isConsortiaEnv = stripes.hasInterface('consortia');
 
-  const { isLoading: isLoadingLinkSuggestions, fetchLinkSuggestions } = useLinkSuggestions();
+  const isRequestToCentralTenantFromMember = applyCentralTenantInHeaders(location, stripes, marcType);
```

Comment:
These 3 lines are duplicated a lot to pass a single param to hooks. Let's add that header inside each of the request hooks

Reply:
Occasionally in the same mode (e.g. Derive) we need to make the same request both [with](https://github.com/folio-org/ui-quick-marc/blob/38df596ec717c12430b487f5b0f7177a9ec76518/src/QuickMarcEditor/QuickMarcEditorContainer.js#L132) and [without](https://github.com/folio-org/ui-quick-marc/blob/38df596ec717c12430b487f5b0f7177a9ec76518/src/QuickMarcEditor/utils.js#L151) central tenant header. 
I think it will be easier to sustain further if we keep this at the top level.

Reply:
Fixed